java/ingest/ingest-batcher-submitter/src/main/java/sleeper/ingest/batcher/submitter/IngestBatcherSubmitterLambda.java

```diff
@@ -96,7 +96,8 @@ public void handleMessage(String json, Instant receivedTime) {
             return;
         }
         requests.forEach(request -> {
-            LOGGER.info("Adding {} to store", request.getFile());
+            LOGGER.info("Storing ingest request for file {} with size {}B to table {}",
+                    request.getFile(), request.getFileSizeBytes(), request.getTableName());
```

Comment:
Maybe we should use a human-readable format for the file size? That applies to the rest of the ingest batcher logs as well though.

We could move the method ClientUtils.formatBytes to the core.

Reply:
Or log both the actual size and the human-readable size: `with size 123456B (123MB)`?